Automattic/woocommerce-payments
# Left align the connect page contents

Fixes #2271 

#### Changes proposed in this Pull Request

This PR removes `text-align:center` style to left-align the contents on the page.


#### Testing instructions

1. Clone this repository and run `npm start`
2. Navigate to the Payments page and confirm that the page is left-aligned.


![Screen Shot 2021-06-18 at 1 56 21 PM](https://user-images.githubusercontent.com/4723145/122615385-4b4f4200-d03d-11eb-8a61-bcda9ab726f2.jpg)
